hudi-common/src/main/java/org/apache/hudi/common/table/read/KeyBasedHoodieFileGroupRecordBuffer.java

```diff
@@ -40,15 +41,18 @@
 import java.util.Iterator;
 import java.util.Map;
 
+import static org.apache.hudi.common.engine.HoodieReaderContext.INTERNAL_META_PARTITION_PATH;
+import static org.apache.hudi.common.engine.HoodieReaderContext.INTERNAL_META_RECORD_KEY;
+
 /**
  * A buffer that is used to store log records by {@link org.apache.hudi.common.table.log.HoodieMergedLogRecordReader}
  * by calling the {@link #processDataBlock} and {@link #processDeleteBlock} methods into a record key based map.
  * The records from the base file is accessed from an iterator object. These records are merged when the
  * {@link #hasNext} method is called.
  */
-public class HoodieKeyBasedFileGroupRecordBuffer<T> extends HoodieBaseFileGroupRecordBuffer<T> {
+public class KeyBasedHoodieFileGroupRecordBuffer<T> extends BaseHoodieFileGroupRecordBuffer<T> {
```

Comment:
Similarly, remove `Hoodie` in the class name for simplicity
```suggestion
public class KeyBasedFileGroupRecordBuffer<T> extends BaseHoodieFileGroupRecordBuffer<T> {
```

Reply:
Done